apps/ejabberd/include/mongoose_api.hrl

```diff
@@ -0,0 +1,17 @@
+-include("mongoose_commands.hrl").
+%% backend state for HTTP API
+-record(http_api_state, {allowed_methods, bindings, parameters, command_category, entity = admin, opts = []}).
+%% Error messages
+-define(ARGS_LEN_ERROR, "Bad parameters length.").
+-define(ARGS_SPEC_ERROR, "Bad name of the parameter.").
+-define(BODY_MALFORMED, "The request body is malformed.").
+-define(COMMANDS_ENGINE, mongoose_commands).
```

Comment:
Why do we need this? I think usuing `mongoose_commands` explicitly will be more readable.